src/vaadin-custom-field.html

```diff
@@ -150,6 +150,15 @@
               observer: '__invalidChanged'
             },
 
+            /**
+             * Specifies that the custom field has a value.
+             */
+            hasValue: {
+              type: Boolean,
+              reflectToAttribute: true,
+              computed: '__computeHasValue(value)'
```

Comment:
Is the `computed` only reason to have `hasValue` exposed as a property?

Wondering if we should instead use observer for `value` and then toggling the `has-value` attribute only.

Reply:
Hmm yeah, maybe it would make more sense. I'll convert this to just toggle the attribute in an observer.

First I just copied this part from `vaadin-text-field` where it is a property (though there it is not even computed).

Reply:
Done.